src/main/java/clients/VehicleTechnicalRecordsClient.java

```diff
@@ -97,6 +97,7 @@ private Response callGetVehicleTechnicalRecords(String searchIdentifier) {
         Response response = given().filters(new BasePathFilter())
                 .contentType(ContentType.JSON)
                 .pathParam("searchIdentifier", searchIdentifier)
+                .queryParam("status", "provisional_over_current")
```

Comment:
This change looks a bit strange - do we know we don't want current records?

Reply:
This was changed with Craig on the call to fix one of the tests and get the endpoint returned.